app/core/api/urls.py

```diff
@@ -16,6 +17,7 @@
 router.register(r"sponsor-partners", SponsorPartnerViewSet, basename="sponsor-partner")
 router.register(r"faqs", FaqViewSet, basename="faq")
 router.register(r"faqs-viewed", FaqViewedViewSet, basename="faq-viewed")
+router.register(r"location", LocationViewSet, basename="location")
```

Comment:
The first argument should be "locations". It's what's used for the endpoint prefix like /api/locations/